index.css

```diff
@@ -0,0 +1,125 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-weight: 700;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+ul {
+  list-style: none;
+}
+
+li {
+  display: flex;
+  margin-block: 0.5em;
+  justify-content: space-between;
+  margin-inline: 2.5em;
+  align-items: center;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-section {
+  display: none;
+  min-width: 30%;
+}
+
+.contact {
+  display: none;
+}
+
+.navBar {
```

Comment:
Optional: You can add the rule `pointer: cursor` to indicate the options in your navbar are clickable.